lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -96,8 +100,19 @@ def safe_method(method_name, node)
           end
         end
 
+        def check_localtime(node)
+          p = node
+          while !p.nil? && p.send_type?
+            break if extract_method(p) == :localtime
+            p = p.parent
```

Comment:
The meaning `parent` makes sense here, but is kind of confusing above. And I'm generally not fond of single-letter variable names.